less/pages/home.less

```diff
@@ -1,56 +1,84 @@
-@quote-symbol-size: 5rem;
-
-.case-studies {
-  padding-bottom: 3rem;
-
-  .quote-symbol:before {
-    content: '\201C';
-    margin-left: -0.15em;
-  }
-
-  .quote-symbol {
-    border-radius: 50%;
-    background-color: #f36b21;
-    color: white;
-    display: inline-block;
-    height: @quote-symbol-size;
-    font-family: Georgia, serif;
-    font-size: @quote-symbol-size;
-    padding-top: 0.05em;
-    width: @quote-symbol-size;
+.blog-section {
+  margin: 3rem 0;
+  h2 {
+    margin-bottom: 3rem;
   }
+}
 
-  @media (min-width: @screen-sm-min) {
-    .row {
-      align-items: stretch;
-      display: flex;
+#featured-post {
```

Comment:
I was thinking there should only be **one** featured post on a page so I used `id` instead of `class`.  I guess over specific selector might not be a good idea so I'm happy to change it into `class`.